hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -260,4 +337,51 @@ public int hashCode() {
     result = 31 * result + prevCommit.hashCode();
     return result;
   }
+
+  public static class PerfStat implements Serializable {
```

Comment:
lets rename this to `RuntimeStats` on second thoughts, more aptly captures intent to group everything that is about some metrics during the execution of the write?

Reply:
Sure, RuntimeStats sounds fine, will do.